_episodes/02-01-shell.md

```diff
@@ -8,7 +8,7 @@ questions:
 objectives:
 - "Provide hints on efficient use of the collected shell history of commands"
 - "Explain how to make shell scripts more robust and less dangerous"
-keypoints:
+keypoints: 
 - "A command line shell is a powerful tool and learning additional
```

Comment:
Typically before doing actual commit I run `git diff` to inspect my changes, and then possibly adjust so I do not introduce unrelated changes, e.g. such a trailing space.